e2-studio-star-rx72n-firmware/libs/rx_spi_link/src/rx_spi_link.c

```
@@ -0,0 +1,731 @@
+ * @param[in]     tx_payload Payload to transmit (may be FEC-encoded)
+ * @param[in]     tx_len     Payload length in bytes
+ * @param[in]     attempt    Attempt number (0 = first, >0 = retransmit)
+ *
+ * @return rx_err_t Result of attempt
+ * @retval k_rx_ok ACK received (success)
+ * @retval k_rx_err_protocol_error NACK received (retry needed)
+ * @retval k_rx_err_timeout Timeout (retry needed)
+ * @retval Other SPI send errors
+ */
+static rx_err_t internal_send_attempt(rx_spi_link_t*  link,
+                                      rx_frame_type_t type,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'internal_send_attempt' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)